src/pages/commits/BuildVariantAccordion.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styled from "@emotion/styled";
+import { uiColors } from "@leafygreen-ui/palette";
+import { Disclaimer } from "@leafygreen-ui/typography";
+
+const { gray } = uiColors;
+
+interface Props {
+  buildVariants: {
+    displayName: string;
+  }[];
+}
+export const BuildVariantAccordion: React.FC<Props> = ({ buildVariants }) => (
+  <ColumnContainer>
+    {buildVariants.map(({ displayName }) => (
+      <AccordionContainer key={displayName}>{displayName}</AccordionContainer>
+    ))}
+  </ColumnContainer>
+);
+
+const AccordionContainer = styled(Disclaimer)`
```

Comment:
Maybe Body would work better here? Since you are overriding the disclaimers font size.

Reply:
Good idea!